consensus/src/chained_bft/block_storage/block_store.rs

```diff
@@ -265,21 +270,23 @@ impl<T: Payload> BlockStore<T> {
 
         // Reconfiguration rule - if a block is a child of pending reconfiguration, it needs to be empty
         // So we roll over the executed state until it's committed and we start new epoch.
-        let output = if parent_block.compute_result().has_reconfiguration() {
-            ProcessedVMOutput::new(
-                vec![],
-                parent_block.output().executed_trees().clone(),
-                parent_block.output().validators().clone(),
+        let state_compute_result = if parent_block.compute_result().has_reconfiguration() {
+            StateComputeResult::new(
+                parent_block.compute_result().root_hash(),
+                parent_block.compute_result().frozen_subtree_roots().clone(),
+                parent_block.compute_result().num_leaves(),
+                parent_block.compute_result().validators().clone(),
+                vec![], /* compute_status */
+                vec![], /* transaction_info_hashes */
```

Comment:
would it make sense to make this a tuple via say a zip operation or something?

Reply:
They are not always of the same length, dude